app/javascript/app/components/sectors-agriculture/countries-context/land-area/land-area-selectors.js

```diff
@@ -0,0 +1,217 @@
+import { createSelector, createStructuredSelector } from 'reselect';
+import {
+  getSelectedCountry,
+  getSelectedYear
+} from '../countries-contexts-selectors';
+
+const AVAILABLE_COLORS_FOR_AGRICULTURAL = [
+  '#0677B3',
+  '#009CC6',
+  '#00B4D2',
+  '#045480'
+];
+
+const AVAILABLE_COLORS_FOR_OTHERS = ['#CACCD0', '#999B9E'];
+
+const TOTAL_INDICATOR = 'share_in_land_area_1';
+const NON_AGRICULTURE_INDICATORS = [
+  'share_in_land_area_3',
+  'share_in_land_area_4'
+];
+const AGRICULTURE_INDICATOR = 'share_in_land_area_2';
+
+const AGRICULTURE_SHARE_AREAS = [
```

Comment:
You can identify indicators in metadata array by 'indicator' field for this endpoint [(documentation)](https://github.com/Vizzuality/climate-watch/blob/6e8c915a911052f5eb4d7b3a14cc986ad1e4960b/docs/data_explorer.md#areas), please check if you can use it :)
![screenshot from 2019-01-25 10-40-37](https://user-images.githubusercontent.com/15097138/51741053-b1625580-208d-11e9-807e-569c00947c88.png)